packages/airnode-node/src/api/index.ts

```diff
@@ -27,7 +27,7 @@ export function buildOptions(payload: ApiCallPayload): adapter.BuildRequestOptio
   const { type, config, aggregatedApiCall } = payload;
   const { endpointName, parameters, oisTitle } = aggregatedApiCall;
 
-  const ois = config.ois.find((o) => o.title === oisTitle)!;
+  const ois = config.ois.find((o) => o.title === oisTitle)! as OIS;
```

Comment:
For some reason I was getting TS errors unless I did this. Not sure if it's caused by a different zod version, but I think someone added these in the past as well (probably for a similar reason).